redfish-exporter/README.md

```diff
@@ -1,6 +1,6 @@
 # AMD Redfish Exporter
 
-The AMD Redfish Exporter is a Go server that subscribes to events and telemetry from servers with Redfish service enabled. It exposes this data on a Prometheus-compatible metrics route for scraping. The exporter also integrates with SLURM to drain nodes when critical hardware events are detected.
+The AMD Redfish Exporter is a Go server that subscribes to events and telemetry from servers with Redfish service enabled. It exposes this data on a Prometheus-compatible metrics route for scraping. The exporter also integrates with SLURM to drain nodes when critical hardware events are detected. This exporter is tested on Slurm version `23.02.7` with REST API version `v0.0.39`
```

Comment:
Redundant? https://github.com/nod-ai/ADA/pull/44/files#diff-e915fa7cad28b3f4423667bcc3a07e88ecbd3dbae46bcb21bbf4ae5da5097e08R97

Reply:
I wanted to clarify the tested version at the beginning of the README for better transparency. I’ll go ahead and keep this update.